contracts/periphery/contracts/s2s/SmartChainXApp.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.6.0;
+
+import "./SmartChainXLib.sol";
+
+abstract contract SmartChainXApp {
+    struct Config {
+        // config for dispatch call
+        address dispatchAddress;
+        bytes2 dispatchCallIndex;
+        // config for market fee
+        address storageAddress;
+    }
+
+    struct BridgeConfig {
+        bytes storageKeyForMarketFee;
```

Comment:
Storage key is dynamic bytes?

Reply:
Yes, it will change with different market fee module